datamade/how-to
# Fatima/cookiecutter sass

## Overview

This PR adds Bootstrap and Sass as dependencies in `package.json`, adds the `develop` script to `package.json`, and adds a `static/scss/custom.scss` file with the boostrap style import to facilitate overriding Bootstrap styles with Sass on apps we set up with the Django and Wagtail cookiecutter boilerplates. It also adds a bit of documentation and helpful links in the `docker/templates` README.

Handles #308 
Handles #319 

### Notes

## Testing Instructions

* Create an example Django app using the cookiecutter and run it using `docker-compose up --build`
* Check a default bootstrap style for your Django app, e.g., navbar color
* Use the Sass file to update the bootstrap style you took note of. Run the develop script on your running container using `docker-compose exec app npm run-script develop`
* Check your browser again and make sure the new style is applied
* make some changes to JavaScript and try to commit them; make sure linting works as expected
* Repeat the above steps for a Wagtail app
